docs/docs/installation.md

```diff
@@ -4,9 +4,11 @@ sidebar_position: 2
 
 # Installation
 
+Selecting the right installation method for the `hydra-node` depends on your requirements and technical preferences. Below are the primary methods to get started quickly and efficiently.
+
 ### Docker
 
-The quickest way to get a `hydra-node` installed and running is to use our Docker images:
+For rapid deployment, Docker offers the most straightforward method to install and run `hydra-node`. Use the following commands to pull the latest Docker image and display the help options to verify the installation:
```

Comment:
Why do we need to explain / paraphrase the command coming after? I'd prefer short and concise over lengthy text.

Reply:
This should work @oduameh :

The quickest way to install and run a `hydra-node` is by using Docker images: